camera_matrix: [890.696167 ,0.000000   ,678.131240 ,0.000000,
                0.000000   ,912.169617 ,365.652024 ,0.000000,
                0.000000   ,0.000000   ,1.000000   ,0.000000]

rlc:  [  0.9994938, -0.0314104, -0.0050614,
   0.0287995,  0.9608312, -0.2756338,
   0.0135209,  0.2753485,  0.9612494 ]

tlc: [-0.0085 ,-0.0 ,-0.15]



